server/dao/user.js

```diff
@@ -13,6 +13,16 @@ class UserDao {
     return user;
   };
 
+  getUserViaEmail = async (email) => {
```

Comment:
This isn't related to this PR, but I'm thinking that the other method in this file should maybe be renamed to `getUserViaID` or `getUserById`. (I can't recall what the common/standard naming convention is - if the current naming scheme follows convention, then disregard this)

Reply:
Yes. That's better. I would probably change that before the end of this sprint.